Python/compile.c

```diff
@@ -563,8 +563,16 @@ compiler_unit_free(struct compiler_unit *u)
 }
 
 static int
-compiler_add_static_attribute_to_class(struct compiler *c, PyObject *attr)
+compiler_maybe_add_static_attribute_to_class(struct compiler *c, expr_ty e)
 {
+    assert(e->kind == Attribute_kind);
+    expr_ty attr_value = e->v.Attribute.value;
+    if (attr_value->kind != Name_kind ||
+        e->v.Attribute.ctx == Load ||
```

Comment:
```suggestion
        e->v.Attribute.ctx != Store ||
```

I think it best to stick to stores, and ignore deletions. I doubt it will make any difference in practice and it is easier to document.